WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AddProductName/AddProductNameWithAIView.swift

```diff
@@ -0,0 +1,225 @@
+                                TextEditor(text: $viewModel.productNameContent)
+                                    .bodyStyle()
+                                    .foregroundColor(.secondary)
+                                    .padding(insets: Layout.messageContentInsets)
+                                    .frame(minHeight: Layout.minimumEditorHeight, maxHeight: .infinity)
+                                    .focused($editorIsFocused)
+
+                                Divider()
+                                    .frame(height: Layout.dividerHeight)
+                                    .foregroundColor(Color(.separator))
+
+                                suggestNameField
+                                    .padding(insets: Layout.suggestButtonInsets)
+                            }
+                            .overlay(
+                                RoundedRectangle(cornerRadius: Layout.cornerRadius).stroke(Color(.brand))
+                            )
+
+                            // Placeholder text
+                            placeholderText
+                        }
+                    }
+
+                    usePackagePhoto
+                }
+            }
+            .padding(insets: Layout.insets)
+        }
+        .safeAreaInset(edge: .bottom) {
+            VStack {
+                // CTA to continue to next screen.
+                continueButton
+                    .padding()
+            }
+            .background(Color(uiColor: .systemBackground))
+        }
+    }
+}
+
+private extension AddProductNameWithAIView {
+    var suggestNameField: some View {
+        HStack {
+            // Suggest a name
+            Button {
+                viewModel.didTapSuggestName()
+            } label: {
+                HStack(alignment: .top) {
+                    Image(uiImage: .sparklesImage)
+                        .renderingMode(.template)
+                        .resizable()
+                        .foregroundColor(Color(.brand))
+                        .frame(width: Layout.sparkleIconSize * scale, height: Layout.sparkleIconSize * scale)
+
+                    Text(Localization.suggestName)
+                        .foregroundColor(Color(.brand))
+                        .bodyStyle()
+                }
+            }
+            Spacer()
+        }
+    }
+
+    var placeholderText: some View {
+        Text(Localization.placeholder)
+            .foregroundColor(Color(.placeholderText))
+            .bodyStyle()
+            .padding(insets: Layout.placeholderInsets)
+            // Allows gestures to pass through to the `TextEditor`.
+            .allowsHitTesting(false)
+            .renderedIf(viewModel.productNameContent.isEmpty)
+    }
+
+    var usePackagePhoto: some View {
+        HStack {
+            // Use package photo
+            Button {
+                viewModel.didTapUsePackagePhoto()
+            } label: {
+                HStack(spacing: Layout.UsePackagePhoto.spacing) {
```

Comment:
🔍  The stack should be aligned with `.top` to be consistent with the suggest name button.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/10723/commits/1cc98856c600afc8aa3947b447909056aabd67b0